lib/proxy.js

```diff
@@ -137,12 +137,13 @@ function buildResultingMiddleware (host, options, stream) {
     var reqOpts = {
       url     :  URL.format(proxyUrl),
       method  : req.method,
-      headers : _.defaults(options.headers || {}, req.headers),
+      // headers : _.defaults(options.headers || {}, req.headers),
+      headers : options.headers,
```

Comment:
I'm not sure theres an extensible way to handle them on our end other than having a set of common apis we have default configs for (as we discussed) but thats more of a post publication thing. 

My goal with this is to have users be able to run against multiple 3rd party API's without writing handlers for them. The can handle headers here if they want, but I thought the proxy should be as unobtrusive as possible. I think a straight proxy would cause less confusion